l10n_ch_fds_upload_sepa/model/fds_postfinance_historical_sepa.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Swiss Postfinance File Delivery Services module for Odoo
+#    Copyright (C) 2014 Compassion CH
+#    @author: Nicolas Tran
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp import models, fields
+
+
+class fds_postfinance_historical_sepa(models.Model):
+    ''' Add historical sepa to the model fds.postfinance.account
+    '''
+    _name = 'fds.postfinance.historical.sepa'
+
+    fds_account_id = fields.Many2one(
+        comodel_name='fds.postfinance.account',
+        string='FDS account id',

```

Comment:
We might prefer "FDS account" `id` is only there to say it's a relation, we don't want to display it to the user.